simulation/simulation_interface/proto/geometry_msgs.proto

```diff
@@ -1,58 +1,65 @@
 syntax = "proto3";
 
 /**
- * Protobuf package to define geometry_msgs in ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
-*/
+ * Protobuf package to define geometry_msgs in
+ * ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
+ */
 package geometry_msgs;
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Point type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Point type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
+ **/
```

Comment:
Malformed indentation

Reply:
fixed in f6866faa0ee81aa8172e74b4322c48adfbbce10e